cmd/wfx-viewer/colors/colors.go

```diff
@@ -0,0 +1,124 @@
+package colors
+
+/*
+ * SPDX-FileCopyrightText: 2024 Siemens AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Author: Michael Adler <michael.adler@siemens.com>
+ */
+
+import (
+	"image/color/palette"
+
+	"github.com/siemens/wfx/generated/model"
+	"gopkg.in/go-playground/colors.v1"
+)
+
+const (
+	DefaultFgColor = "white"
+	DefaultBgColor = "#000000"
```

Comment:
Hm, isn't `black` defined as well if `white` is?

Reply:
Yeah, one should think so... I tried it and it didn't work :)